### ISSUE TITLE
[docs] Showcase notebook failing to render

### ISSUE THREAD

**Issue by IAlibay** [OpenFreeEnergy/openfe#843]
```
/home/docs/checkouts/readthedocs.org/user_builds/openfe/checkouts/841/docs/tutorials/showcase_notebook.nblink:1159: WARNING: Literal block expected; none found.
```

**Comment by IAlibay** [comment]
Will fix in #841

**Comment by mikemhenry** [comment]
Will actually fix in https://github.com/OpenFreeEnergy/openfe/pull/733
